src/site/content/en/blog/hands-on-portals/index.md

```diff
@@ -267,4 +286,4 @@ Another important thing to know is that Portals can be used in cross-origin navi
 
 ## Feedback welcome
 
-The Portals proposal is still in the early stages, so not everything is working yet. (That's why it's behind an experimental flag.) That said, it's ready for experimentation in Chrome Canary. Feedback from the community is crucial to the design of new APIs, so please try it out and tell us what you think! You can check the current limitations on the [Chromium bug tracker](https://bugs.chromium.org/p/chromium/issues/detail?id=957836). If you have any feature requests or feedback, please head over to the [WICG GitHub repo](https://github.com/WICG/portals/issues).
+Portals are ready for experimentation in Chrome Canary and via the [origin trial](https://www.chromium.org/blink/origin-trials/portals). Feedback from the community is crucial to the design of new APIs, so please try it out and tell us what you think! You can check the current limitations on the [Chromium bug tracker](https://bugs.chromium.org/p/chromium/issues/detail?id=957836). If you have any feature requests or feedback, please head over to the [WICG GitHub repo](https://github.com/WICG/portals/issues).
```

Comment:
I don't know that crbug.com/957836 is that useful going forward.

Reply:
I agree, the crbug.com/957836 is stale. I feel there is no documentation around summarizing all the current limitations so should we just remove this for now, or do we want to change the link to something like: https://bugs.chromium.org/p/chromium/issues/list?q=component:Blink%3EHTML%3EPortal ?

Reply:
Option 1: remove. I personally feel this option would be okay since I now see many of the core limitations already listed up in the [OT explainer](https://www.chromium.org/blink/origin-trials/portals) e.g. [_I've noticed some differences between what Chrome does and what the explainer and/or spec say. What gives?_](https://www.chromium.org/blink/origin-trials/portals#:~:text=I've%20noticed%20some%20differences%20between%20what%20Chrome%20does%20and%20what%20the%20explainer%20and/or%20spec%20say.%20What%20gives?)
```suggestion
Portals are ready for experimentation in Chrome Canary and via the [origin trial](https://www.chromium.org/blink/origin-trials/portals). Feedback from the community is crucial to the design of new APIs, so please try it out and tell us what you think! If you have any feature requests or feedback, please head over to the [WICG GitHub repo](https://github.com/WICG/portals/issues).
```

Option 2: change link to the crbug list for `component:Blink>HTML>Portal`
```suggestion
Portals are ready for experimentation in Chrome Canary and via the [origin trial](https://www.chromium.org/blink/origin-trials/portals). Feedback from the community is crucial to the design of new APIs, so please try it out and tell us what you think! You can check the current limitations on the [Chromium bug tracker](https://bugs.chromium.org/p/chromium/issues/list?q=component:Blink%3EHTML%3EPortal). If you have any feature requests or feedback, please head over to the [WICG GitHub repo](https://github.com/WICG/portals/issues).
```
Other suggestions? @jakearchibald @jeremyroman